### ISSUE TITLE
Renaming a model in text editing

### ISSUE THREAD

**Issue by lochel** [OpenModelica/OpenModelica#7196]
### Description
Renaming a SSP model in text editing causes problems. OMEdit doesn't recognize the new model name and still tries to query the model using its old name. As a result, the model is destroyed and cannot be recovered, see the screenshot below. 

It is very easy to fix though, becase `oms_loadSnapshot` returns already the new cref which should be used.

> oms_status_enu_t oms_loadSnapshot(const char* cref, const char* snapshot, char** newCref);

### Screenshots
![image](https://user-images.githubusercontent.com/8457302/108469093-f7e40780-7287-11eb-94b6-a4c46230dcbf.png)

### Version
```bash
> omc --version && OMSimulator --version
OMCompiler v1.18.0-dev.42+g7646132712
OMSimulator v2.1.1.post24-gd2f351b-linux
```

**Comment by lochel** [comment]
@adeas31 Can you please fix this in OMEdit? The API returns already the new model name and you just need to update your own scope if the new name is different from the old name.

**Comment by adeas31** [comment]
@arun3688 is this still valid after your recent updates.

**Comment by arun3688** [comment]
@adeas31 yes this is still valid, it is not possible to change the model name via text editior, you can change the model name from the library tree item
